### ISSUE TITLE
Automated CircleCI deploys to dev space often fail on the first attempt

### ISSUE THREAD

**Issue by jseppi** [18F/calc#1723]
A subsequent manually-triggered rebuild from the CircleCI web UI is typically successful.

The failure always happens on the deploy of `calc-rqscheduler`, after `calc` and `calc-rqworker` successfully deploy.

<details>
<summary>Click to expand typical logged output of the failure (from <a href="https://circleci.com/gh/18F/calc/408">https://circleci.com/gh/18F/calc/408</a>)</summary>

```
Traceback (most recent call last):
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 484, in connect
           sock = self._connect()
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 541, in _connect
           raise err
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 529, in _connect
           sock.connect(socket_address)
       ConnectionRefusedError: [Errno 111] Connection refused
       
       During handling of the above exception, another exception occurred:
       
       Traceback (most recent call last):
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/client.py", line 667, in execute_command
           connection.send_command(*args)
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 610, in send_command
           self.send_packed_command(self.pack_command(*args))
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 585, in send_packed_command
           self.connect()
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 489, in connect
           raise ConnectionError(self._error_message(e))
       redis.exceptions.ConnectionError: Error 111 connecting to REDACTED. Connection refused.
       
       During handling of the above exception, another exception occurred:
       
       Traceback (most recent call last):
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 484, in connect
           sock = self._connect()
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 541, in _connect
           raise err
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 529, in _connect
           sock.connect(socket_address)
       ConnectionRefusedError: [Errno 111] Connection refused
       
       During handling of the above exception, another exception occurred:
       
       Traceback (most recent call last):
         File "/tmp/app/manage.py", line 12, in <module>
           execute_from_command_line(sys.argv)
         File "/tmp/app/docker_django_management.py", line 251, in execute_from_command_line
           execute_from_command_line(argv)
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
           utility.execute()
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/django/core/management/__init__.py", line 338, in execute
           django.setup()
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/django/__init__.py", line 27, in setup
           apps.populate(settings.INSTALLED_APPS)
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/django/apps/registry.py", line 116, in populate
           app_config.ready()
         File "/tmp/app/data_capture/apps.py", line 47, in ready
           for job in scheduler.get_jobs():
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/rq_scheduler/scheduler.py", line 247, in get_jobs
           start=offset, num=length)
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/client.py", line 1814, in zrangebyscore
           return self.execute_command(*pieces, **options)
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/client.py", line 673, in execute_command
           connection.send_command(*args)
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 610, in send_command
           self.send_packed_command(self.pack_command(*args))
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 585, in send_packed_command
           self.connect()
         File "/tmp/contents311201214/deps/0/python/lib/python3.6/site-packages/redis/connection.py", line 489, in connect
           raise ConnectionError(self._error_message(e))
       redis.exceptions.ConnectionError: Error 111 connecting to REDACTED. Connection refused.
       **ERROR**  !     Error while running '$ python /tmp/app/manage.py collectstatic --noinput'.
              See traceback above for details.
       
              You may need to update application code to resolve this error.
              Or, you can disable collectstatic for this application:
       
                 $ cf set-env <app> DISABLE_COLLECTSTATIC 1
       
              https://devcenter.heroku.com/articles/django-assets
       **ERROR** Error handling collectstatic: exit status 1
Failed to compile droplet: Failed to run finalize script: exit status 12
Exit status 223
```
</details>

**Comment by jseppi** [comment]
hrm, the latest build+deploy that happened (https://circleci.com/gh/18F/calc/442) was successful. I wonder if it was a python buildpack thing or something else under-the-hood that got corrected recently? I'll dig again if we see it again...

**Comment by jseppi** [comment]
Oh, it could have also been that the last build corrected a small issue with our manifest files (#1650), so maybe that is what fixed this issue as well.

**Comment by jseppi** [comment]
This stopped happening 🎉, likely because #1650 corrected the issue.
